apps/video/js/video.js

```diff
@@ -299,14 +302,9 @@ function toggleVideoControls(e) {
   // We cannot change the visibility state of video contorls when we are in
   // picking mode.
   if (!pendingPick) {
-    if (!controlShowing) {
-      // If control not shown, tap any place to show it.
-      setControlsVisibility(true);
-      e.cancelBubble = true;
-    } else if (e.originalTarget === dom.videoControls) {
-      // If control is shown, only tap the empty area should show it.
-      setControlsVisibility(false);
-    }
+    e.cancelBubble = !controlShowing;
+    // If control not shown, tap any place to show it.
+    setControlsVisibility(!controlShowing);
```

Comment:
Given the code change here, this comment is no longer relevant. My suggestion is either no comment or something mentioning toggling the visibility of the controls.